code/processes/rdb.q

```diff
@@ -100,6 +105,10 @@ endofday:{[date;processdata]
 	a:{(x;raze exec {(enlist x)!enlist((#);enlist y;x)}'[c;a] from meta x where not null a)}each tables`.;
 	/-save and wipe the tables
 	writedown[hdbdir;date];
+        /-creates new changeset if this is a finspace application
+        if[.finspace.enabled;
+                    changeset:.finspace.createChangeset[.finspace.database];
```

Comment:
Not a fan of variable setting inside if statements as it creates a scenario where sometimes a variable exists, sometimes not at run time